seqr/views/apis/dataset_api_tests.py

```diff
@@ -14,6 +14,13 @@
 SV_INDEX_NAME = 'test_new_sv_index'
 NEW_SAMPLE_TYPE_INDEX_NAME = 'test_new_index'
 ADD_DATASET_PAYLOAD = json.dumps({'elasticsearchIndex': INDEX_NAME, 'datasetType': 'VARIANTS'})
+MAPPING_PROPS_NO_SAMPLES = {
+    "samples_num_alt_1": {"type": "keyword"},
+}
+MAPPING_PROPS_WITH_SAMPLES = {
+    "samples": {"type": "keyword"},
+    "samples_num_alt_1": {"type": "keyword"},
```

Comment:
you will never have an index with both of these fields, so this is not realistic data. you should just have `samples` here